### ISSUE TITLE
Out of date docs - Custom Connector For Azure Protected Functions

### ISSUE THREAD

**Issue by kathryn-avanade** [MicrosoftDocs/BusinessApplicationPlatform-Connectors-public#491]
Create a custom connector for azure protected functions seems to be out of date, the portal experience doesn't reflect the screenshots or instructions which makes this article hard to follow.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cd26cace-2743-6fb0-daea-30969b9941d1
* Version Independent ID: 8c578f39-2571-60ba-e250-e69f9d5d2b2c
* Content: [Create a custom connector for Azure AD protected Azure Functions](https://docs.microsoft.com/en-us/connectors/custom-connectors/create-custom-connector-aad-protected-azure-functions#feedback)
* Content Source: [docs/custom-connectors/create-custom-connector-aad-protected-azure-functions.md](https://github.com/MicrosoftDocs/BusinessApplicationPlatform-Connectors/blob/live/docs/custom-connectors/create-custom-connector-aad-protected-azure-functions.md)
* Service: **connectors**
* GitHub Login: @devkeydet
* Microsoft Alias: **marcsc**

**Comment by v-aangie** [comment]
@kathryn-avanade  - Thanks for contacting us. If you’re still experiencing any issues, please submit a message on one of our community forums. To help you choose the forum that best describes your issue, go to [Provide feedback or get help for connectors | Microsoft Docs](https://docs.microsoft.com/en-us/connectors/custom-connectors/provide-feedback) We’re closing this issue.
